packages/net/tcpconnection.pony

```diff
@@ -138,6 +149,17 @@ actor TCPConnection
     @pony_os_peername[Bool](_fd, ip)
     ip
 
+  fun ref expect(qty: USize = 0) =>
+    """
+    A `received` call on the notifier must contain exactly `qty` bytes. If
+    `qty` is zero, the call can contain any amount of data. This has no effect
+    if called in the `sent` notifier callback.
```

Comment:
Further, I've never implemented SSL but is throwing away data really the right behavior?  Is the incoming TCP guaranteed to be framed properly in each read?